tests/airdrop.ts

```diff
@@ -227,67 +231,4 @@ describe('airdrop', () => {
     assert(Number((await getAccount(provider.connection, source, 'single')).amount) === claimAmountTwo.toNumber() + claimAmountThree.toNumber());
   });
 
-  it('GovernanceE2E', async () => {
```

Comment:
Been done somewhere else?

Reply:
Actual code is removed and replaced with a TODO for now. In the next couple of days will fix to make it compatible with the flip, put it back in the sdk, and replace the testing